starter_code/js/main.js

```diff
@@ -0,0 +1,35 @@
+
+
+$( document ).ready(function() {
+    console.log( "document loaded" );
+
+    $("#bottom-cta").click(function(event){
+      event.preventDefault();
+      console.log("click prevented");
+    });
+
+    $(".readmore").on('click', function() {
```

Comment:
@Defy1x great use of the console log! I would have continued using the .click here but .on works too. great job overall! 👍